packages/fxa-content-server/tests/functional/send_sms.js

```diff
@@ -202,48 +216,49 @@ const suite = {
     'empty phone number': function() {
       return this.remote
         .then(openPage(SEND_SMS_URL, selectors.SMS_SEND.HEADER))
-        .then(click(selectors.SMS_SEND.SUBMIT))
-        .then(testElementExists(selectors.SMS_SEND.PHONE_NUMBER_TOOLTIP))
-        .then(
-          testElementTextInclude(
-            selectors.SMS_SEND.PHONE_NUMBER_TOOLTIP,
-            'required'
-          )
-        );
+        .then(testElementDisabled(selectors.SMS_SEND.SUBMIT));
     },
 
     'invalid phone number (too short)': function() {
       return this.remote
         .then(openPage(SEND_SMS_URL, selectors.SMS_SEND.HEADER))
         .then(type(selectors.SMS_SEND.PHONE_NUMBER, '2134567'))
-        .then(click(selectors.SMS_SEND.SUBMIT))
-        .then(testElementExists(selectors.SMS_SEND.PHONE_NUMBER_TOOLTIP))
-        .then(
-          testElementTextInclude(
-            selectors.SMS_SEND.PHONE_NUMBER_TOOLTIP,
-            'invalid'
-          )
-        );
+        .then(testElementDisabled(selectors.SMS_SEND.SUBMIT));
```

Comment:
I still think we should have at least one test with a full number that validates on the client-side and hits the `invalid phone number` server tooltip error (like `12231231234`) but these changes look good 👍

Reply:
I might be misunderstanding, but that's tested [here](https://github.com/mozilla/fxa/pull/4454/files#diff-fe73cbe23907b9f163e8e8c80dd49cf9R257)? `12231231234` is a valid phone number, so should pass, but we test against a mixed-character type value.

Reply:
Hmm, it passes the client-side validation and the button becomes enabled, but the server still returns this error. I think the jQ plugin is still a crude check.

![image](https://user-images.githubusercontent.com/13018240/76251687-78475f00-6215-11ea-8fa0-32cc7243c68a.png)

Reply:
Updated to test this failure!